include/tesseract/baseapi.h

```diff
@@ -531,6 +532,28 @@ class TESS_API TessBaseAPI {
    * as UTF8 and must be freed with the delete [] operator.
    */
   char *GetUTF8Text();
+  
+  size_t GetNumberOfTables();
+  
+  /// Return the i-th table bounding box coordinates
+  ///
+  ///Gives the (top_left.x, top_left.y, bottom_right.x, bottom_right.y)
+  /// coordinates of the i-th table.
+  std::tuple<int,int,int,int> GetTableBoundingBox(
+    unsigned i///< Index of the table, for upper limit \see GetNumberOfTables()
+  );
+  
+  /// Get bounding boxes of the rows of a table
+  /// return values are (top_left.x, top_left.y, bottom_right.x, bottom_right.y)
+  std::vector<std::tuple<int,int,int,int> > GetTableRows(
```

Comment:
Can we use some existing struct with four ints? Some RECT or some other tess class?